js/tests/unit/dom/manipulator.spec.js

```diff
@@ -60,14 +60,30 @@ describe('Manipulator', () => {
       expect().nothing()
     })
 
-    it('should get all data attributes', () => {
-      fixtureEl.innerHTML = '<div data-test="js" data-test2="js2" ></div>'
+    it('should get all data attributes, without bs prefixed as well', () => {
+      fixtureEl.innerHTML = '<div data-bs-toggle="tabs" data-bs-target="#element" data-another="value"></div>'
 
       const div = fixtureEl.querySelector('div')
 
       expect(Manipulator.getDataAttributes(div)).toEqual({
-        test: 'js',
-        test2: 'js2'
+        bsToggle: 'tabs',
+        bsTarget: '#element',
+        another: 'value',
+        toggle: 'tabs',
+        target: '#element'
```

Comment:
sorry if I wasn't clear but for me should just expect that here:
```js
{
  bsToggle: 'tabs',
  bsTarget: '#element',	        
  another: 'value',
}
```

any thoughts here @XhmikosR ?

Reply:
@Johann-S **bs** is removed from the attributes because it is not present in the configuration of the plugins.
For example:
https://github.com/twbs/bootstrap/blob/e0b8fcdf899aa1c25fe2ddf050452f7451ed0cdd/js/src/carousel.js#L42-L49
Ref: https://github.com/twbs/bootstrap/commit/1232ed355a7ee9a201f5d7c14ffe4b19ede41e98#commitcomment-40771099

Reply:
Ok understood 👌